src/styles/blocks/header.scss

```
@@ -0,0 +1,92 @@
+.header {
+  background-image: url(../images/photos/header.png);
+  background-position: 50% 70%;
+  background-repeat: no-repeat;
```

https://developer.mozilla.org/ru/docs/Web/CSS/background
```suggestion
  background: ......
```

---

I think the semantics in my option is much more understandable than urs.